.github/workflows/ec2_runner_jobs.yml

```diff
@@ -5,6 +5,10 @@ on:
       job_type:          {required: true, type: string, description: Selects the steps to enable}
       label:             {type: string, description: the label of the machine that was create by the start-runner job needed by the stop-runner job}
       ec2-instance-id:   {type: string, description: the ec2-instance-id of the machine that was create by the start-runner job needed by the stop-runner job}
+      image-id:          {default: ami-08b85498b04e33a4a, type: string, description: ID of the Amazon Image that should be used as a base for the runner}
```

Comment:
My comment about the magic numbers is more about what's changed in the actual content, i.e. why do we have to switch from `ami-07ee6ba755b7f58f6` to `ami-08b85498b04e33a4a`, not what these fields mean.

Reply:
The newer image has some new dependencies installed/upgrade for existing ones.